Makefile

```diff
@@ -24,6 +24,10 @@ endif
 define all-charts
 	@for chart in $$(ls -1 $(CHARTS_DIR)); do \
 		CHART=$$chart make $(MAKE_OPTS) $(1) ; \
+		exit_code=$$? ; \
```

Comment:
you might be able to do `make whateva || exit $$?`

Reply:
🙏 Yes!  Thank you; updated.   Love the simplification.